packages/main/src/plugin/onboarding-registry.ts

```diff
@@ -142,4 +143,34 @@ export class OnboardingRegistry {
       }
     });
   }
+
+  /**
+   * checkIdsReadibility checks that the IDs of the steps of the onboarding
+   * respect specific rules, so they are easily readable in Telemetry.
+   *
+   * In case of a rule not respected, a warning is displayed in the console.
+   */
+  checkIdsReadability(extension: AnalyzedExtension, onboarding: Onboarding) {
+    const warn = (msg: string) => {
+      console.warn(`${extension.name} Onboarding: ${msg}`);
+    };
+    onboarding.steps.forEach(step => {
+      const id = step.id;
+      const isCommand = !!step.command || !!step.component;
```

Comment:
Here, I'm not sure if it is reasonable to consider a screen with a "component" to be a command.